hugr-core/src/hugr/views/sibling_subgraph.rs

```diff
@@ -277,9 +277,6 @@ impl SiblingSubgraph {
     ///
     /// The subgraph signature will be given by signature of the node.
     pub fn from_node(node: Node, hugr: &impl HugrView) -> Self {
-        // TODO once https://github.com/CQCL/portgraph/issues/155
```

Comment:
this comment is no longer true since the two methods have different functionality

If we can extend the functionality of `try_from_nodes` that would be great: #1766